gr-analog/python/analog/qa_sig_source.py

```diff
@@ -198,7 +198,7 @@ def test_freq_msg(self):
         self.assertAlmostEqual(src.frequency(), 1.0)
 
         frequency = 3.0
-        src._post(pmt.to_pmt('freq'), pmt.from_double(frequency))
+        src._post(pmt.to_pmt('cmd'), pmt.from_double(frequency))
```

Comment:
We can keep this test and add another it to test setting frequency, phase, amplitude, and offset using key/value pairs.